testing/php7-ssh2/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Andy Postnikov <apostnikov@gmail.com>
+# Maintainer: Valery Kartel <valery.kartel@gmail.com>
```

Comment:
Yes, please contact him (e.g. via email) and ask if he agree. Or you can just specify yourself as the package maintainer.